### PR TITLE
2024-01-18 meeting page

### LINKED ISSUE #16: 2024-01-18 @ 17:00
Steps to take when setting up a new meeting...

+ [x] **Date** : 2024-01-18
+ [x] **Time** : 17:00 to 18:30
+ [x] **Venue** : Requested [Diamond G04 Workroom 1](https://sites.google.com/sheffield.ac.uk/pooledroomdirectory/teaching-spaces/the-diamond/g04-workroom-1) ([SHEF 2312 0936](https://shef.topdesk.net/tas/public/ssp/content/detail/incident?unid=6ced4b13d9124bb0a4f014d1a7b55192)) 2023-12-04
+ [x] **Request details from @RobertASmith** : Request a summary of the talk and a [speaker biography](https://darkpeakanalytics.com/#team).
+ [x] **Request details from @W-Mohammed** : Request a summary of the talk and a speaker [speaker biography](https://darkpeakanalytics.com/#team).
+ [x] **Website Post** : See #18.
+ [ ] **Announce on Meetup Mailing List** : Announce the meeting on the [Meetup SheffieldR User Group](https://www.meetup.com/sheffieldr-sheffield-r-users-group/) mailing list.

### PR DESCRIPTION
**Pull request by ns-rse**
First draft of the 2024-01-18 page.

Closes #16

**NB** Please ignore the pre-commit CI failure, I'll introduce this linting in a separate PR after discussion as to whether its something you are happy to integrate and use.

### CODE REVIEW

File: posts/2024-01-18-jan-meetup/index.qmd [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+image: image.png
+excerpt_separator: <!--more-->
+---
+
+SheffieldR User Group is back in January 2024 with a Health Economics special and two talks from Robert Smith and Wael
+Mohammed of [Dark Peak Analytics](https://darkpeakanalytics.com/) and the University of Sheffield who will talk us
+through two of their recent publications. Rob will show us how to automate Health Economic Evaluation with R without
+sharing sensitive data and Wael will guide us through a tutorial on R packaging for cost-effectiveness models.
+
+Meeting is 2024-01-08 17:00-18:30 in The Diamond G04 Workroom 1. Sign up
```
Comment (tdjames1) [inline posts/2024-01-18-jan-meetup/index.qmd]: Correct date:

```suggestion
Meeting is 2024-01-18 17:00-18:30 in The Diamond G04 Workroom 1. Sign up
```

N.B. This needs to be fixed on the Meetup event, too.

File: posts/2024-01-18-jan-meetup/index.qmd [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+[here](https://www.meetup.com/sheffieldr-sheffield-r-users-group/events/297758319/).
+
+If you have any questions or inquires please get in touch with [Grace Accad](mailto:g.accad@sheffield.ac.uk) or [Neil
```
Comment (tdjames1) [inline posts/2024-01-18-jan-meetup/index.qmd]: Amend spelling and correct typo:
```suggestion
If you have any questions or enquiries please get in touch with [Grace Accad](mailto:g.accad@sheffield.ac.uk) or [Neil
```

File: posts/2024-01-18-jan-meetup/index.qmd [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+Shephard](mailto:n.shephard@sheffield.ac.uk)/
```
Comment (tdjames1) [inline posts/2024-01-18-jan-meetup/index.qmd]: Correct punctuation:
```suggestion
Shephard](mailto:n.shephard@sheffield.ac.uk).
```

**Review by tdjames1 — CHANGES_REQUESTED** [review]
Hi @ns-rse, just picking up a couple of minor typos here.

**Comment by ns-rse** [comment]
Thanks @tdjames1 for the attentive eye for detail, pretty shoddy I mis-typed the date!

**Review by tdjames1 — APPROVED** [review]
Looking good now. I have updated the date on the meetup event blurb!

**Pull request merged**